README.md

```diff
@@ -1,50 +1,92 @@
 [![Build Status](https://travis-ci.org/kubernetes-csi/external-provisioner.svg?branch=master)](https://travis-ci.org/kubernetes-csi/external-provisioner)
 
-# Kubernetes external provisioner that works with CSI volumes
+# CSI provisioner
 
-This is an example external provisioner for Kubernetes which provisions using CSI Volume drivers..  It's under heavy development, so at this time README.md is notes for the developers coding.  Once complete this will change to something user friendly.
+The external-provisioner is a sidecar container that dynamically provisions volumes by calling `ControllerCreateVolume` and `ControlerDeleteVolume` functions of CSI drivers. It is necessary because internal persistent volume controller running in Kubernetes controller-manager does not have any direct interfaces to CSI drivers.
 
-# Build
+## Overview
+The external-provisioner is an external controller that monitors `PersistentVolumeClaim` objects created by user and creates/deletes volumes for them. Full design can be found at Kubernetes proposal at [container-storage-interface.md](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md)
 
-```bash
-make csi-provisioner
-```
+## Compatibility
 
-# Test
+This information reflects the head of this branch.
 
-## Start Kubernetes
+| Compatible with CSI Version                                                                | Container Image                        | Min K8s Version |
+| ------------------------------------------------------------------------------------------ | ---------------------------------------| --------------- |
```

Comment:
Change this to 1.14 (due to topology being on by default)

Reply:
fixed (but the image version is now wrong)

Reply:
Let's pre-call it 1.1.0?

Reply:
actually how about just canary?

I think official cut versions we can keep in csi-docs

Reply:
Hmm, I'd like to have **single** place where we have image versions for all supported Kubernetes releases. If it's in csi-docs, then we should link it from here. It will simplify release process a bit.

Reply:
Removed version from the image.